### PR TITLE
Add 'Find me on Github' Link to Navbar

### PR DESCRIPTION
**Pull request by Ricardo-Silva91**
Proposed fix for issue #51 
changes/additions:
 - added the Github logo image file to the 'public directory'
 - added the Github link to NavigationBar
 - created new style for 'navbar-link-right' class elements.

### CODE REVIEW

**Comment by tmobaird** [comment]
@Ricardo-Silva91 thanks a ton for this PR! I'll review this in a little 👍

File: src/components/NavigationBar.jsx [hunk 1]
```diff
@@ -12,6 +13,7 @@ class NavigationBar extends Component {
         <Link to="/about" className="navbar-link" activeClassName="active">About</Link>
         {' | '}
         <Link to="/contributors" className="navbar-link" activeClassName="active">Contributors</Link>
+        <a href="https://github.com/tmobaird/i-want-to-contribute" className="navbar-link navbar-link-right">Find me on Github <img alt="" src={githubLogo}></img></a>
```
Comment (tmobaird) [inline src/components/NavigationBar.jsx]: Let's change the `alt` attribute to `Github`

File: src/styles/navigation-bar.css [hunk 2]
```diff
@@ -18,3 +18,14 @@
...
+
+.navbar-link-right {
+  float: right;
+  margin-right: 1.5%;
+  color: black;
+}
+
+.navbar-link-right:hover {
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.61);
```
Comment (tmobaird) [inline src/styles/navigation-bar.css]: I think we should change this to opacity. So we only need to rely on the base color from `navbar-link-right`, then simply change the opacity on hover to make both the text and image appear to be lighter... Ex:

```css
.navbar-link-right {
  float: right;
  margin-right: 1.5%;
  color: black;
}
.navbar-link-right:hover {
  text-decoration: none;
  opacity: 0.7;
}
```

File: src/components/NavigationBar.jsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { Link, IndexLink } from 'react-router';
 import { Row } from 'react-bootstrap';
 import '../styles/navigation-bar.css';
+import githubLogo from '../../public/GitHub-Mark-32px.png';
```
Comment (tmobaird) [inline src/components/NavigationBar.jsx]: This should be changed to `'../images/Github-Mark-32px.png'`

**Comment by tmobaird** [comment]
@Ricardo-Silva91 also, go ahead and add your name to the proper locations (or new locations) in [CONTRIBUTORS.md](https://github.com/tmobaird/i-want-to-contribute/blob/master/CONTRIBUTORS.md) and [contributors.json](https://github.com/tmobaird/i-want-to-contribute/blob/master/src/utils/contributors.json) if you'd like. If you don't feel like disclosing your location, feel free to add an `Anonymous` location! Btw, the second link will allow your name to show up in the live app!

**Comment by Ricardo-Silva91** [comment]
@tmobaird Hello again.
Just pushed the changes you asked for, let me know if you need any more edits.

**Comment by tmobaird** [comment]
Hey @Ricardo-Silva91, the changes were perfect! I merged this in via 87a0a8e. Thanks a ton for the contribution, awesome to see contributions coming from Portugal!! 👍 🍩 

Feel free to help out with anything else on this project if you'd like!

**Pull request closed without merging**